backend/core/src/listings/listings.service.ts

```diff
@@ -107,6 +115,90 @@ export class ListingsService {
     }
   }
 
+  public async listIncludeExternal(
+    bloomJurisdictions: string[],
+    params: ListingsQueryParams
+  ): Promise<ListingIncludeExternalResponse> {
+    return {
+      local: await this.list(params),
+      external: await this.listExternal(bloomJurisdictions, params),
+    }
+  }
+
+  private async listExternal(
+    bloomJurisdictions: string[],
+    params: ListingsQueryParams
+  ): Promise<JurisdictionIdToExternalResponse> {
+    if (bloomJurisdictions == null || bloomJurisdictions.length == 0) {
```

Comment:
is it possible that `bloomJurisdictions` is null? Is it not always a string array, per the type system?

Reply:
This is my first time working with typescript but I think that the default is that it is still nullable even though it doesn't explicitly say so (FWIW I added this bc it was throwing an exception) -_-